### PR TITLE
ADCM-1846 Stabilization of UI test host reset config

### PR DESCRIPTION
**Pull request by Sealwing**
One of full tests flacked, added retry, added wait in cases when save config process was a problem, rearranged steps

### CODE REVIEW

**Review by annademina — DISMISSED** [review]

**Review by dgusakov — DISMISSED** [review]

**Review by annademina — APPROVED** [review]

File: tests/ui_tests/app/page/common/configuration/page.py [hunk 1]
```diff
@@ -70,6 +73,17 @@ def get_input_value(self, adcm_test: str, is_password: bool = False):
         )
         return self.find_element(template(adcm_test)).get_property("value")
 
+    @allure.step('Check input of field "{adcm_test}" has value "{expected_value}"')
+    def assert_input_value_is(self, expected_value: str, adcm_test: str, is_password: bool = False):
```
Comment (dgusakov) [inline tests/ui_tests/app/page/common/configuration/page.py:77]: Why adcm_test is a string and not a webElement?

Comment (Sealwing) [reply tests/ui_tests/app/page/common/configuration/page.py:77]: It's a string that is passed to format TemplateLocator to generate an actual Locator.
It follows the signature of `get_input_value` method of the Configuration page.

**Review by dgusakov — APPROVED** [review]

**Comment by Sealwing** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @Sealwing! Build has been launched!

**Review by dgusakov — APPROVED** [review]

**Review by annademina — APPROVED** [review]

**Pull request merged**